chrony/osfamilymap.yaml

```diff
@@ -0,0 +1,77 @@
+
+# -*- coding: utf-8 -*-                                                                                             
```

Comment:
```diff
-
-# -*- coding: utf-8 -*-                                                                                             
+# -*- coding: utf-8 -*-
```

Blank line and trailing space to be resolved.

Reply:
I have removed that line and fix that ridiculous trailing whitespace.